.env.example

```diff
@@ -20,6 +20,7 @@ ARCHIVE_NODE_ethereum=https://eth-mainnet.nodereal.io/v1/<YOUR_KEY_HERE>
 #ARCHIVE_NODE_zksyncmainnet=https://open-platform.nodereal.io/<YOUR_KEY_HERE>/zksync
 #ARCHIVE_NODE_opsepolia=https://sepolia.optimism.io
 #ARCHIVE_NODE_opmainnet=https://opt-mainnet.nodereal.io/v1/<YOUR_KEY_HERE>
+ARCHIVE_NODE_basesepolia=https://base-sepolia.blastapi.io/<YOUR_KEY_HERE>
```

Comment:
Could we set here `https://sepolia.base.org` for sepolia? For arbitrum sepolia, where nodereal is not an option, we set the "official" RPC. We can do the same for base sepolia, setting the "official" RPC endpoint (according to [their docs](https://docs.base.org/docs/network-information/))